src/components/Brick.js

```diff
@@ -14,19 +14,19 @@ import {
 import { ERROR } from '../utils/evalUtils'
 
 class Brick extends Component {
-  constructor(props) {
+  constructor (props) {
     super(props)
 
     this.startDrag = this.startDrag.bind(this)
   }
 
-  startDrag(mouseEvent) {
+  startDrag (mouseEvent) {
     const { handleMouseDown, id, position } = this.props
 
     handleMouseDown(id, mouseEvent, position)
   }
 
-  render() {
+  render () {
```

Comment:
There are 2 rules that I don't like. The first is this space (useless in my opinion)